src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -788,7 +787,12 @@ export function getCustomizedConfiguration(chartOptions: IChartOptions, chartCon
         return merge(config, configurator(chartOptions, config));
     }, {});
 
-    const labelConfigData = getLabelsConfiguration(chartOptions, chartConfig);
+    const chartConfigData = [
```

Comment:
what about to keep getLabelsConfiguration and getStackingConfiguration in array of configurators and only add third param:
`function getLabelsConfiguration(chartOptions: any, {}: any, config?: IChartConfig) {...`
and 
`return merge(config, configurator(chartOptions, config, chartConfig));`